collectors/networkviewer.plugin/README.md

```diff
@@ -0,0 +1,85 @@
+# networkviewer.plugin
+
+This plugin uses eBPF to bring connection information from the kernel ring to user ring. 
+
+The default behavior of this plugin is to monitor all connections from the privates IPs defined inside the RFC 1918 
+(https://tools.ietf.org/html/rfc1918) to any destination, this default behavior can be changed with the configuration file
+`networkviewer.conf` .
+
+## Only Linux for while
+
+This plugin for while works only on Linux, but we are monitoring the development of eBPF on other Operate Systems. For 
+Linux there is also a limitation, the collector won't work for kernel verson smaller than `4.11.0` .
+
+## Kprobe and tracing file system
+
+For this collector works it will be necessary your kernel to be compiled with support to `k[ret]probe`, case this was not
+done, the collector won't be able to monitor the traffic of data.
```

Comment:
I would suggest phrasing this as:

'This collector requires the kernel to be built with support for kprobes (`CONFIG_KPROBES=y`).'

Reply:
I confess I tried to avoid give kernel options in the documentation, because I was thinking this could not be clear for all users, but I agree that somehow they will need to do this case their distribution does not have Kprobe enabled.